src/schema/artist/statuses.js

```diff
@@ -45,12 +50,22 @@ const ArtistStatusesType = new GraphQLObjectType({
     },
     contemporary: {
       type: GraphQLBoolean,
-      resolve: ({ id }) => {
-        return total(`related/layer/contemporary/artists`, {
-          exclude_artists_without_artworks: true,
-          artist: [id],
-          size: 0,
-        }).then(count => count > 0)
+      resolve: (
+        { id },
+        options,
+        request,
+        { rootValue: { relatedContemporaryArtistsLoader } }
+      ) => {
+        return totalViaLoader(
+          relatedContemporaryArtistsLoader,
+          {},
+          {
+            exclude_artists_without_artworks: true,
+            artist: [id],
+            size: 0,
+            total_count: 1,
```

Comment:
Why do these last two still need to be included?

Reply:
Oh yea they don't!